src/node/node_state.h

```diff
@@ -1548,8 +1548,8 @@ namespace ccf
         for (const auto& interface : config.node_info_network.rpc_interfaces)
         {
           sans.push_back(
-            {interface.rpc_address.hostname,
-             is_ip(interface.rpc_address.hostname)});
+            {interface.public_rpc_address.hostname,
+             is_ip(interface.public_rpc_address.hostname)});
```

Comment:
Come to think of it, we should mention this in the CHANGELOG, it is user-visible.